sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -39,7 +34,7 @@ describe("EventHub Receiver #RunnableInBrowser", function(): void {
       env[EnvVarKeys.EVENTHUB_NAME],
       "define EVENTHUB_NAME in your environment before running integration tests."
     );
-    partitionIds = await client.getPartitionIds();
+    partitionIds = await client.getPartitionIds({});
```

Comment:
The object is still optional though right?

Reply:
It's optional in the public facing interfaces (EventHubProducerClient, EventHubConsumerClient). It's required here since EventHubClient became our "lower level" interface and I didn't want to accidentally fail to pass the options on.